exception_reporter.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'telemetry_logger'
   spec.add_dependency 'dependency'
-  spec.add_dependency 'null_attr'
+  spec.add_dependency 'subst_attr'
```

Comment:
It does *something*, but I don't recall what.  You're right that it doesn't quite manage the dependency properly.  I think it might not do anything...  

Yay for custom dependency management systems... =P